### PR TITLE
Add generic functions

### PR DESCRIPTION
**Pull request by JKrymarys**

### CODE REVIEW

File: POiD/basic_operations.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import cv2
+import utils as utils
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
import utils
```

File: POiD/basic_operations.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def increase_image_brightness(img):
+    img_height = utils.get_image_height(img)
+    img_width = utils.get_image_width(img)
+    new_img = []
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
```

File: POiD/basic_operations.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    new_img = img.copy()
+
+    y = 0
+    x = 0
+    c = 0 
+
+    b = 50
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
```

File: POiD/basic_operations.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    # while y < img_height:
+    for y in range(img_height):
+        for x in range(img_width):
+            for c in range(3):
+                if img[y][x][c] + b > 255 :
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
                if img[y][x][c] + brightness > 255 :
```

File: POiD/basic_operations.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                    new_img[y][x][c] = 255
+                else:
+                    new_img[y][x][c] = img[y][x][c] + b
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
                    new_img[y][x][c] = img[y][x][c] + brightness
```

File: POiD/basic_operations.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+    return new_img
+
+def decrease_image_brightness(img):
+    img_height = utils.get_image_height(img)
+    img_width = utils.get_image_width(img)
+    new_img = []
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
```

File: POiD/basic_operations.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        for x in range(img_width):
+            for c in range(3):
+                if img[y][x][c] + b > 255 :
+                    new_img[y][x][c] = 255
+                else:
+                    new_img[y][x][c] = img[y][x][c] + b
+
+    return new_img
+
+def decrease_image_brightness(img):
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
def decrease_image_brightness(img, brightness):
```

File: POiD/basic_operations.py [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    img_height = utils.get_image_height(img)
+    img_width = utils.get_image_width(img)
+    new_img = []
+    new_img = img.copy()
+
+    y = 0
+    x = 0
+    c = 0 
+
+    b = 50
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
```

File: POiD/basic_operations.py [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+    # while y < img_height:
+    for y in range(img_height):
+        for x in range(img_width):
+            for c in range(3):
+                if img[y][x][c] - b < 0 :
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
                if img[y][x][c] - brightness < 0 :
```

File: POiD/basic_operations.py [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+                    new_img[y][x][c] = 0
+                else:
+                    new_img[y][x][c] = img[y][x][c] - b
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
                    new_img[y][x][c] = img[y][x][c] - brightness
```

File: POiD/core.py [hunk 11]
```diff
@@ -0,0 +1,19 @@
+
+import cv2 
+import basic_operations as basic_operations
+import utils as utils
+
+try:
+    img = utils.load_image('cat.jpg')
+    utils.display_image('original',img)
+
+    utils.display_image('brighter', basic_operations.increase_image_brightness(img))
```
Comment (paulinapwyrwas) [inline POiD/core.py]:
```suggestion
    utils.display_image('brighter', basic_operations.increase_image_brightness(img, 50))
```

File: POiD/core.py [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+    utils.display_image('darker', basic_operations.decrease_image_brightness(img))
```
Comment (paulinapwyrwas) [inline POiD/core.py]:
```suggestion
    utils.display_image('darker', basic_operations.decrease_image_brightness(img, 50))
```

File: POiD/basic_operations.py [hunk 13]
```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import cv2
+import utils as utils
+
+
+def increase_image_brightness(img):
```
Comment (paulinapwyrwas) [inline POiD/basic_operations.py]:
```suggestion
def increase_image_brightness(img, brightness):
```

**Review by paulinapwyrwas — CHANGES_REQUESTED** [review]

**Comment by JKrymarys** [comment]
@paulinapwyrwas  plz review again :)

**Review by paulinapwyrwas — APPROVED** [review]

**Pull request merged**